### DISCUSSION TITLE
CommandLine issue

### DISCUSSION THREAD

**Discussion by nici22** [winsiderss/systeminformer#1981] [Questions / Answers] [answered]
I checked svchost.exe on both Task Manager and System Informer 
2 of them have no CommandLine value on TaskManager, but available on SystemInformer
Is there any reason for that?

**Comment by diversenok** [comment]
Most likely it's Task Manager failing to retrieve information about protected processes. You can check if that's the case by enabling the Protection column in System Informer or checking the down-left corner of the General tab for given processes.
![image](https://github.com/winsiderss/systeminformer/assets/30962924/f4468c7b-d88a-463e-b2ea-4406dc1d8efc)

**Reply by dmex** [reply]
The commandline of every process can be queried via infoclass and Windows supports protected processes as it only requires query_limited access.

**Reply by nici22** [reply]
Okay, thank you for replying

**Comment by dmex** [comment, marked as answer]
System Informer removes null terminators from the command line which are used by some services.
